### PR TITLE
NAS-130604 / 25.04 / CPU model widget

### PR DESCRIPTION
**Pull request by AlexKarpov98**
**Changes:**

New widget.

**Testing:**
See ticket.

Screenshot 2024-08-15 at 09 42 50

### Downstream

|Affects         |Reasoning
|----------------|-------------------------------
|Documentation   | New widget.

### CODE REVIEW

File: src/app/pages/dashboard/widgets/system/widget-cpu-model/widget-cpu-model.definition.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { marker as T } from '@biesbjerg/ngx-translate-extract-marker';
+import { WidgetCategory } from 'app/pages/dashboard/types/widget-category.enum';
+import { dashboardWidget } from 'app/pages/dashboard/types/widget-component.interface';
+import { SlotSize } from 'app/pages/dashboard/types/widget.interface';
+import { WidgetCpuModelComponent } from 'app/pages/dashboard/widgets/system/widget-cpu-model/widget-cpu-model.component';
+
+export const cpuModelWidget = dashboardWidget({
+  name: T('CPU Model'),
+  component: WidgetCpuModelComponent,
+  category: WidgetCategory.SystemInfo,
```
Comment (undsoft) [inline src/app/pages/dashboard/widgets/system/widget-cpu-model/widget-cpu-model.definition.ts:10]: Should be under CPU category.

**Review by undsoft — APPROVED** [review]
Looks epyc.

File: src/app/pages/dashboard/widgets/system/widget-cpu-model/widget-cpu-model.definition.ts (hunk 1 above)
Comment (AlexKarpov98) [reply src/app/pages/dashboard/widgets/system/widget-cpu-model/widget-cpu-model.definition.ts:10]: it was there initially, then I thought it's better to move it to System 😄

Comment (AlexKarpov98) [reply src/app/pages/dashboard/widgets/system/widget-cpu-model/widget-cpu-model.definition.ts]: ✅ 
Screenshot 2024-08-15 at 18 15 09

**Pull request merged**